README.md

```diff
@@ -1,87 +1,86 @@
-# biconomy-paymasters
+# Biconomy Paymasters 🌐
 
-### Playground for ERC4337 Paymasters built with :heart_eyes: by Biconomy 
-#### Some examples 
+Welcome to the `biconomy-paymasters` repository! Paymaster contracts enable seamless transaction fee handling in decentralized applications. 🛠️
 
-1. Singleton Verifying Paymaster: Acts as a sponsorship paymaster and lets Dapps manage deposit without deploying a new one for each Dapp.
+## What is Biconomy Paymasters? 🤔
 
-2. Token Paymaster: Extended version of Verifying Paymaster which can accept fees from user by withdrawing ERC20 tokens
+Biconomy Paymasters are smart contracts that abstract away the complexity of gas fees for end-users. By utilizing these contracts, developers can offer their users gasless transactions or the ability to pay for gas in ERC20 tokens. 🚀
```

Comment:
can just limit to gas abstraction. then talk about kind of paymaster we already have or we plan to have with gas abstraction (how) aspect of each